src/styles/main.scss

```diff
@@ -1,3 +1,235 @@
-body {
+$blue_color: #00acdc;
+$dark-blue_color: #060b35;
+$card-items_width: 166px;
+
+@mixin card_size {
+  width: 198px;
+  height: 406px;
+}
+
+@mixin star_size {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.header,
+.nav__list,
+.nav__item,
+.nav__link,
+.is-active {
+  display: flex;
+}
+
+.header {
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  width: 100%;
+  background-color: white;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  box-sizing: border-box;
+}
+
+.nav__list {
+  font-family: "Roboto", sans-serif;
+  list-style: none;
+  text-transform: uppercase;
+  font-weight: 500;
+  font-size: 12px;
+  padding: 0;
   margin: 0;
 }
+
+.nav__item {
+  position: relative;
+  align-items: center;
+  cursor: pointer;
+  margin-left: 20px;
+  height: 60px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  flex-direction: column;
+  text-decoration: none;
+  justify-content: center;
+  color: $dark-blue_color;
+  height: 60px;
+}
+
+.nav__item:hover > .nav__link {
+  color: $blue_color;
+}
+
+.is-active {
+  color: $blue_color;
+}
+
+.is-active::after,
+.nav__link:active::after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  height: 4px;
+  background-color: $blue_color;
+  border-radius: 8px;
+}
+
+.card_container {
+  display: flex;
+  flex-wrap: wrap;
+  margin-top: 51px;
+  width: 944px;
```

Comment:
add max-width

Reply:
also make it dynamic
add css variables for card width, margin between cards and calculate width from them

Reply:
+